http/http_socket_handler.h

```diff
@@ -1,6 +1,4 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
-
-/* http_socket_handler.h                                           -*- C++ -*-
+/* http_socket_handler.h - This file is part of MLDB               -*- C++ -*-
```

Comment:
It's still present below. But I concede the style is not consistent with the rest.